src/main/java/model/OrderRepo.java

```diff
@@ -0,0 +1,44 @@
+package model;
+
+import java.util.*;
+
+public class OrderRepo {
+
+    private Map<Integer, Order> orders = new HashMap<>();
+
+
+    public List<Order> listOfOrders(){
+        ArrayList<Order> orderList = new ArrayList<>(orders.values());
```

Comment:
Saubere, leicht zu lesende Konvertierung - gefällt mir gut!
Man kann argumentieren, dass das in der Lesbarkeit teils besser ist als 
`return new ArrayList<>(...)`

auf jeden fall lieber zu lang als zu kurz! :)